mllib/src/main/scala/org/apache/spark/ml/evaluation/MultilabelClassificationEvaluator.scala

```diff
@@ -96,6 +96,7 @@ class MultilabelClassificationEvaluator (override val uid: String)
         .rdd.map { row =>
         (row.getSeq[Double](0).toArray, row.getSeq[Double](1).toArray)
       }
+    predictionAndLabels.persist()
```

Comment:
Why would persisting here be an advantage? This method doesn't use it more than once

Reply:
It is used mulple times in new MultilabelMetrics to initilize fields.

Reply:
I don't think so, it looks like the summarizer computes the summary stats in one go.
In any event that would be an issue for MultilabelMetrics

Reply:
Sorry, this issue is found in 2.4.3. MultilabelMetrics has changed in 2.4.4.